samples/app/src/main/java/com/example/app/callback/PingOneProtectInitCallback.kt

```diff
@@ -24,12 +24,12 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
 import org.forgerock.android.auth.Logger
-import org.forgerock.android.auth.PingOneProtectInitCallback
+import org.forgerock.android.auth.PingOneProtectInitializeCallback
 import org.forgerock.android.auth.PingOneProtectInitException
 
 @Composable
 fun PingOneProtectInitCallback(
```

Comment:
To align with other, please rename it to PingOneProtectInitializeCallback

Reply:
👍🏻  Renamed...